src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -83,7 +85,41 @@ func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
+func logPanic(msg string) {
+	cfg, err := config.Init(packageVersion)
+	if err != nil {
+		panic("Error initializing config")
+	}
+	if err := logging.Init(); err != nil {
+		panic("Error initializing logging")
+	}
+
+	<-logging.Configure("", "", cfg.InstanceId, version, revisionDate)
+
+	log.Error(msg)
```

Comment:
Yes, the output is "panic". The message passed to the `logPanic` function is exactly the output produced by a normal panic. We could just look for `panic` in the output.